lib/Devel/hdb.pm

```diff
@@ -33,7 +33,12 @@ sub import {
         } elsif ($param =~ m/listenfd:(\d+)/) {
             our $LISTEN_SOCK = IO::Socket::INET->new();
             $LISTEN_SOCK->fdopen($1, 'r');
-        }
+        } elsif ($param =~ m/\Alaunch(:(.+))?/) {
+	    our $LAUNCH = $1 ? $2 : do {
+		require Browser::Open;
```

Comment:
If the require fails (say, because ```Browser::Open``` isn't installed), it should print something to that affect to STDERR with some instructions on how to get it installed.

Reply:
Good thought. I will look into it.